java/ql/src/semmle/code/java/Expr.qll

```diff
@@ -653,11 +653,23 @@ class LongLiteral extends Literal, @longliteral {
 
 /** A floating point literal. For example, `4.2f`. */
 class FloatingPointLiteral extends Literal, @floatingpointliteral {
+  /**
+   * Gets the value of this literal as CodeQL 64-bit `float`. The value will
+   * be parsed as Java 32-bit `float` and then converted to a CodeQL `float`.
```

Comment:
Added this clarify how a value which is not a precise Java `float`, e.g. `1.55555555555f`, will be parsed.
That is, it will have the value `1.5555556` and not `1.55555555555` (which could be represented using CodeQL's `float`).